content/faq/was-ist-openshift-und-wie-funktioniert-es.en.md

```diff
@@ -1,4 +1,4 @@
 ---
 title: What is OpenShift and how does it work?
 ---
-[OpenShift](https://docs.openshift.com/) is the Container platform by Red Hat. Since Version 3 OpenShift relies on proven Open Source projects such as [Docker](http://www.docker.com/) and [Kubernetes](http://kubernetes.io/).
+[OpenShift](https://docs.openshift.com) is the Container platform by Red Hat. Since Version 4 OpenShift relies on proven Open Source projects such as [OCI](https://opencontainers.org) and [Kubernetes](http://kubernetes.io/). OCI compatible container images can be generated using [Buildah](https://buildah.io/) or [Docker](http://www.docker.com/).
```

Comment:
```suggestion
[OpenShift](https://docs.openshift.com) is the Container platform by Red Hat. Since Version 4 OpenShift relies on proven Open Source projects such as [OCI](https://opencontainers.org) and [Kubernetes](https://kubernetes.io/). OCI compatible container images can be built using [Buildah](https://buildah.io/) or [Docker](https://www.docker.com/).
```